test/functional/Java8andUp/testng.xml

```diff
@@ -387,5 +387,10 @@
 			<class name="org.openj9.test.floatsanity.TestFactory" />
 		</classes>
 	</test>
+	<test name="reflect_getDeclaredFields">
+		<classes>
+			<class name="org.openj9.test.reflect.GetDeclaredFieldsTests" />
+		</classes>
+	</test>
```

Comment:
If the plan is to run the test with generalTest, then `<class name="org.openj9.test.reflect.GetDeclaredFieldsTests" />` should be added under `<test name="generalTest">`.
With the current change, there is no cmd to execute this test.